vector/src/main/java/im/vector/app/features/home/ShortcutsHandler.kt

```diff
@@ -130,8 +137,15 @@ class ShortcutsHandler @Inject constructor(
         if (isRequestPinShortcutSupported) {
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
                 context.getSystemService<ShortcutManager>()
-                        ?.let {
-                            it.disableShortcuts(it.pinnedShortcuts.map { pinnedShortcut -> pinnedShortcut.id })
+                        ?.pinnedShortcuts
+                        ?.takeIf { it.isNotEmpty() }
+                        ?.map { pinnedShortcut -> pinnedShortcut.id }
+                        ?.let { shortcutIdsToDisable ->
+                            ShortcutManagerCompat.disableShortcuts(
+                                    context,
+                                    shortcutIdsToDisable,
+                                    stringProvider.getString(R.string.shortcut_disabled_reason_sign_out)
```

Comment:
to double check, is `session has been signed out` the right message or should it be something to do with the room pin?

Reply:
this fun `clearShortcuts()` is called during sign out so I think the message is OK.
When user click on the (greyed out) pin, this message will be displayed